sdk/table/azure-table/azure/table/_message_encoding.py

```diff
@@ -0,0 +1,150 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=unused-argument
+
+from base64 import b64encode, b64decode
+
+import sys
+import six
+from azure.core.exceptions import DecodeError
+
+from ._shared.encryption import decrypt_queue_message, encrypt_queue_message
+
+
+class MessageEncodePolicy(object):
```

Comment:
@kristapratico I think I saw this file before elsewhere no? Shouldn't this be in _shared?

Reply:
yes should be in _shared